src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj

```diff
@@ -5,7 +5,11 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp1.0</TargetFramework>
     <PreserveCompilationContext>true</PreserveCompilationContext>
-  </PropertyGroup>
+  </PropertyGroup>$if$ ($context$ == WebCore)
```

Comment:
We do not handle these in the CLI today. I don't understand what this is doing in the CLI template.